services/explorer/src/api/types.ts

```diff
@@ -131,3 +131,21 @@ export interface TablePagination<T> {
 
 export type BlockTablePagination = TablePagination<BlockData>;
 export type TxTablePaginate = TablePagination<Transaction>;
+
+export type ToastData = {
+  display: boolean;
+  variant?: 'success' | 'error' | 'warning' | undefined;
+  text?: string;
+};

```

Comment:
Why would this be in the API?

Reply:
Since its being used in https://github.com/Taraxa-project/taraxa-platform/blob/68ae9a2e8e46cb78e607e77dd018d01f74646da4/services/explorer/src/api/sendRequestTokens.faucet.ts#L3 as callback type